integration/daemon/main_test.go

```diff
@@ -0,0 +1,33 @@
+package daemon // import "github.com/docker/docker/integration/daemon"
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/docker/docker/testutil/environment"
+)
+
+var testEnv *environment.Execution
+
+func TestMain(m *testing.M) {
+	var err error
+	testEnv, err = environment.New()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	err = environment.EnsureFrozenImagesLinux(testEnv)
```

Comment:
How do these tests use frozen images? Or any of this `environment/protect` calls actually. The added tests do not seem to use shared daemon at all.

Reply:
Yes, looks like we should be able to remove for now as the current tests don't use the shared daemon.

w.r.t. the `testEnv`, perhaps we can just skip on `! linux` instead.


@aiordache could you have a look at removing the unneeded setup code?